hpx/include/parallel_copy.hpp

```diff
@@ -8,8 +8,7 @@
 #define HPX_PARALLEL_COPY_JUN_28_2014_0827AM
 
 #include <hpx/parallel/algorithms/copy.hpp>
-#include <hpx/parallel/segmented_algorithms/copy.hpp>
-#include <hpx/parallel/container_algorithms/copy.hpp>
+#include <hpx/parallel/container_algorithms.hpp>
 
```

Comment:
What is the rationale of this change? Why include all container algorithms in this header file?